tests/test_pydantic_settings.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
```

Comment:
Would it be better to put this file into `pydantic-settings` repo. It won't work without it anyway.

Reply:
I think this file shouldn't be in `pydantic-settings` repo because it belongs to this repo.
Also, we have same file for `fast-api`

Reply:
👌🏻